src/style.css

```
@@ -0,0 +1,90 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.field {
```

add font-family for .field
use inherit for Inheritance font-family from HTML
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit

```suggestion
.field {
  font-family: inherit;
```